x/ccv/provider/keeper/relay.go

```diff
@@ -343,10 +343,27 @@ func (k Keeper) OnRecvSlashPacket(
 		return ccv.V1Result, nil
 	}
 
-	// Check that chain is launched and the validator belongs to the consumer chain valset
-	if k.GetConsumerPhase(ctx, consumerId) == providertypes.CONSUMER_PHASE_LAUNCHED && !k.IsConsumerValidator(ctx, consumerId, providerConsAddr) {
-		k.Logger(ctx).Error("cannot jail validator %s that does not belong to consumer %s valset",
-			providerConsAddr.String(), consumerId)
+	// check that the chain is launched
+	if k.GetConsumerPhase(ctx, consumerId) != providertypes.CONSUMER_PHASE_LAUNCHED {
+		k.Logger(ctx).Info("cannot jail validator on a chain that is not currently launched",
+			"consumerId", consumerId,
+			"phase", k.GetConsumerPhase(ctx, consumerId),
+			"provider cons addr", providerConsAddr.String(),
+		)
+
+		// drop packet but return a slash ack
```

Comment:
could you write out in what situations this can happen, to have more context here?